src/constants/translations/ua/chat.json

```diff
@@ -1,5 +1,7 @@
 {
-    "message": {
-        "you": "Ви"
-    }
-}
\ No newline at end of file
+  "message": {
+    "you": "Ви"
+  },
+  "yourMessage": "Ви:",
```

Comment:
I think, you can use only "message" for this and add ":" character or you can remove "message" and left "yourMessage", because, I don't see that you used it anywhere

Reply:
Ok, fixed